sktime/classification/feature_based/tests/test_tsfresh_classifier.py

```diff
@@ -209,30 +173,28 @@ def test_tsfresh_classifier_on_basic_motions():
 #     print(']')
 #
 # if __name__ == "__main__":
```

Comment:
Is this staying to help test in the future and so is commented out just so the tests dont run on it, or is it a forgot to remove?